packages/core/src/plugins/injection/index.ts

```diff
@@ -48,107 +43,144 @@ export const getInjectionPlugins = (
         },
     };
 
-    // This plugin happens in 3 steps in order to cover all bundlers:
+    // Create a unique filename to avoid conflicts.
+    const INJECTED_FILE_PATH = `${Math.random().toString().replace('0.', '')}_${INJECTED_FILE}.js`;
```

Comment:
I was afraid of having conflicts with `Date.now()` when multiple builds are launched together (I believe web-ui does that when there are more than one entry).
Never confirmed it though.
Agreed it would be cleaner, I'll try this.